internal/config/config.go

```diff
@@ -59,6 +59,11 @@ func Load() *Config {
 
 	flag.Parse()
 
+	//check if evoBlocksRange is less than 10 otherwise return an error
+	if *evoBlocksRange > 10 {
```

Comment:
`evoBlocksRange` should be limited to 1 and `blocksRange` should be limited to 10. Please, change the default values as well.